kubernetes/cache/ztunnel_dump.go

```diff
@@ -10,9 +14,14 @@ func ztunnelDumpKey(cluster, namespace, pod string) string {
 
 type ZtunnelDumpCache interface {
 	SetZtunnelDump(ztunnelConfig map[string]*kubernetes.ZtunnelConfigDump)
+	GetAllZtunnelDump() map[string]*kubernetes.ZtunnelConfigDump
 	GetZtunnelDump(cluster, namespace, pod string) *kubernetes.ZtunnelConfigDump
 }
 
+func (c *kialiCacheImpl) GetAllZtunnelDump() map[string]*kubernetes.ZtunnelConfigDump {
+	return c.ztunnelConfigStore.Items()
+}
```

Comment:
If you change `SetZtunnelDump` to match `GetZtunnelDump` where it accepts `cluster, namespace, pod` as args then you won't need this method I think.